ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py

```diff
@@ -324,6 +324,38 @@ def test_jax_devicearray_conj(
         on_device=on_device,
     )
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="jax.numpy.array",
+    method_name="imag",
+    dtype_and_x=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("real_and_complex"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid),
```

Valid automatically determines the supported data-types for the function.